javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -742,6 +743,37 @@ public static Tuple0 tuple() {
         return Validation.invalid(error);
     }
 
+    /**
+     * Alias for {@link CharSeq#of(char)}
+     *
+     * @param character A character.
+     * @return A new {@code CharSeq} instance containing the given element
+     */
+    public static CharSeq charSeq(char character) {
+        return CharSeq.of(character);
+    }
+
+    /**
+     * Alias for {@link CharSeq#of(char...)}
+     *
+     * @param characters Zero or more characters.
+     * @return A string containing the given characters in the same order.
+     * @throws NullPointerException if {@code elements} is null
+     */
+    public static CharSeq charSeq(char... characters) {
+        return CharSeq.of(characters);
+    }
+
+    /**
+     * Alias for {@link CharSeq#of(CharSequence)}
+     *
+     * @param sequence {@code CharSequence} instance.
+     * @return A new {@code javaslang.String}
+     */
+    public static CharSeq charSeq(CharSequence sequence) {
```

Comment:
I think we could/should have a general `Object` static factory here also/instead.
It could make sense to construct a `CharSeq` from a number also, e.g. `charSeq(3.14)`
@danieldietrich?